tests/server/bundle_manager_test.py

```diff
@@ -5,9 +5,24 @@
 from codalab.server.bundle_manager import BundleManager
 from codalab.worker.bundle_state import RunResources
 from codalab.bundles import RunBundle
+from codalab.lib.codalab_manager import CodaLabManager
 
 
 class BundleManagerTest(unittest.TestCase):
+    def setUp(self):
```

Comment:
What does this have to do with the main functionality of this PR?

Reply:
When I refactored the tag matching logic to `_get_matched_workers` as a separate function, I noticed that it can't be called from `_filter_and_sort_workers` as `_get_matched_workers` is a class function but `_filter_and_sort_workers` is declared as a static function. Then I searched through our codebase, I found out that the static function `_filter_and_sort_workers` is only used by `BundleManagerTest`. If this is only created for unit testing, I think it might not be necessary to keep only this function as a static function within the entire `BundleManager` class. Moreover, it makes more sense to make it as a class function instead of a static function. So I removed the `staticmethod` declaration for `_filter_and_sort_workers` and added missing settings for `BundleManagerTest`.